features/ticketing/jira-integration/pom.xml

```diff
@@ -7,55 +7,36 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
     <artifactId>jira-troubleticketer</artifactId>
-    <packaging>jar</packaging>
-    <name>OpenNMS Features JIRA Trouble Ticketer</name>
-
-    <repositories>
-        <repository>
-            <id>atlassian-public</id>
-            <name>Atlassian public repo</name>
-            <url>http://maven.opennms.org/content/repositories/atlassian-public/</url>
-        </repository>
-    </repositories>
+    <packaging>bundle</packaging>
+    <name>OpenNMS :: Features :: Ticketing :: JIRA</name>
 
     <build>
-        <plugins>
-            <plugin>
-                <artifactId>maven-assembly-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>attached</goal>
-                        </goals>
-                    </execution>
-                </executions>
-                <configuration>
-                    <descriptorRefs>
-                        <descriptorRef>jar-with-dependencies</descriptorRef>
-                    </descriptorRefs>
-                    <!-- appendAssemblyId>false</appendAssemblyId -->
-                    <tarLongFileMode>posix</tarLongFileMode>
-                </configuration>
-            </plugin>
-        </plugins>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.felix</groupId>
+          <artifactId>maven-bundle-plugin</artifactId>
+          <extensions>true</extensions>
+          <configuration>
+            <instructions>
+              <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
+            </instructions>
+          </configuration>
+        </plugin>
+        <plugin>
+            <groupId>org.opennms.maven.plugins</groupId>
+            <artifactId>features-maven-plugin</artifactId>
+            <configuration>
+                <bundles>
+                    <bundle>mvn:joda-time/joda-time/${jodaTimeVersion}</bundle>
```

Comment:
You do not have to define the bundles manually if they are defined in the dependencies as not provided and are real bundles.